cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/EntityOperatorReconciler.java

```diff
@@ -390,6 +400,24 @@ protected Future<Void> userOperatorSecret(Clock clock) {
                     .map((Void) null);
         }
     }
+    /**
+     * Manages the Entity Operator Network Policies.
+     *
+     * @return  Future which completes when the reconciliation is done
+     */
+    protected Future<Void> networkPolicy() {
+        if (isNetworkPolicyGeneration) {
+            return networkPolicyOperator
+                    .reconcile(
+                            reconciliation,
+                            reconciliation.namespace(),
+                            reconciliation.name() + "-network-policy-entity-operator",
```

Comment:
Similar o the comments earlier => I would use `KafkaResources.entityOperatorDeploymentName(reconciliation.name())` here.